pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -1089,32 +1129,26 @@ func (vs *VSphere) CreateVolume(name string, size int, tags *map[string]string)
 		return "", err
 	}
 
-	if (*tags)["adapterType"] == "" {
-		(*tags)["adapterType"] = LSILogicControllerType
```

Comment:
Other adapterType includes ide and buslogic which old and are not recommended to use. Also, plugin doesn't support ide adapterType.